bt/algos.py

```diff
@@ -493,9 +494,9 @@ class SelectMomentum(AlgoStack):
     """
 
     def __init__(self, n, lookback=pd.DateOffset(months=3),
-                 sort_descending=True):
+                 lag=pd.DateOffset(days=1), sort_descending=True):
```

Comment:
Lag implementation looks good to me. Only thing I would change is the default value to 0 for the lag parameter. This will avoid modifying existing strategies and I think 0 should be the default behavior as per our discussion (#8).

Thanks again for your contributions!
Phil